### DISCUSSION TITLE
Remove type hints from signature help when Type checking mode is set to off

### DISCUSSION THREAD

**Discussion by miron** [microsoft/pylance-release#2766] [Enhancement]
## Environment data

-   Language Server version: v2022.4.3
-   OS and version: Microsoft Windows 11
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.4

## Expected behavior

Original view of signature with very good readability
## Actual behavior

Crammed display of parameter values , type hints, literal ellipses, ... with very bad readability, wordwrapped in a tiny window. 
Other important information like default values is missing.
Because type checking is turned off by default, type hint information makes no sense in that setting.

## Code Snippet / Additional information

Although I knew about type hints already, it took me a month to realize what was going on.
My brain automatically skipped over the signature.

Type hints are great, but not in this format:

![image](https://user-images.githubusercontent.com/109377/166393752-1c71e740-3cbb-4e53-be5e-5b7b55b6470e.png)

Here the original signature as displayed by help():

`sort_values(self, by, axis: 'Axis' = 0, ascending=True, inplace: 'bool' = False, kind: 'str' = 'quicksort', na_position: 'str' = 'last', ignore_index: 'bool' = False, key: 'ValueKeyFunc' = None)`

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
